src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -614,18 +443,19 @@ public void init() {
 
             // Set window title - copy tab title
             StringBinding windowTitle = Bindings.createStringBinding(
-                    () -> libraryTab.textProperty().getValue() + " \u2013 " + FRAME_TITLE,
+                    () -> libraryTab.textProperty().getValue() + " – " + FRAME_TITLE, // not a minus, but codepoint 2013
```

Comment:
I would keep the line above - and just say in the comment that it is "-" :)

Reply:
The compiler replaces the decoded string anyway. The important information is that its not a minus. This is in the comment now.